ts/features/pn/aar/hooks/useSendAarFlowManager.tsx

```diff
@@ -35,6 +42,14 @@ export const useSendAarFlowManager = (): SendAarFlowManager => {
           })
         );
         break;
+      case sendAARFlowStates.notAddresseeFinal:
+        openWebUrl(delegateUrl);
+        break;
+      case sendAARFlowStates.ko:
+        if (isAarErrorRetriable(currentFlowData.errorKind)) {
+          dispatch(setAarFlowState(currentFlowData.previousState));
+        }
```

Comment:
given the new design, I believe that the "retry" logic has been scrapped -- can you update the connected logic?